### DISCUSSION TITLE
Where to request/suggest scripts?

### DISCUSSION THREAD

**Discussion by Slingtwit** [violentmonkey/violentmonkey#1385] [Q&A]
Is there somewhere I can request/suggest a script for someone else to write since I don't have any experience myself?
Doesn't have to be on github.

**Comment by leogama** [comment]
I think **here** is actually a good place to start. But you can also take a look in these places:
* https://openuserjs.org/forum
* https://greasyfork.org/discussions
* https://sleazyfork.org/discussions
* https://www.reddit.com/r/userscripts/

Say what kind of userscript you want. Maybe it already exists or something similar exists and you just need to tweak the code a little bit.
